wit/wadm/client.wit

```diff
@@ -0,0 +1,40 @@
+package wasmcloud:wadm@0.1.0;
+
+interface client {
+    use wasmcloud:oam/types@0.1.0.{oam-manifest};
```

Comment:
Looks like you did end up combining the oam stuff into wadm types, so should this be removed from the wit files?